### DISCUSSION TITLE
How can I run EXPLAIN ANALYZE to test RLS policies?

### DISCUSSION THREAD

**Discussion by pranavbadami** [supabase/supabase#8840] [Questions]
Hey folks, quick question here. I'm trying to mock session data and see the query plan output with RLS enabled. 
```
 begin;
    set local role authenticated;
    select set_config('request.jwt.claim.app_metadata', '{"workspace": "215..."}', true);
    EXPLAIN ANALYZE SELECT * FROM active_users('T02...', date'2021-10-21', date'2021-10-28');
commit;
```

This query runs under the hood but I can never see the query plan getting displayed in my SQL editor (Arctype) or in Supabase's own SQL editor. For example, after a few seconds, I just see `Success. No rows returned`.

How can I actually view the query plan output? Thanks!

**Comment by GaryAustin1** [comment]
Try running without begin;/commit; in the SQL window.

**Comment by steve-chavez** [comment]
Cross posting the solution to RLS performance issues with custom JWT claims: https://github.com/orgs/supabase/discussions/9311#discussioncomment-5690939
